Cargo.toml

```diff
@@ -6,7 +6,6 @@ members = [
   "prqlc/prql-compiler-macros",
   "prqlc/prqlc",
   "prqlc/prql-compiler/examples/compile-files", # An example
-  "prqlc/tests_misc",
```

Comment:
I wasn't confident enough that I was correct to do this myself, but I do agree collapsing this seems good...